blockchain/types/tx_internal_data_serializer_test.go

```diff
@@ -797,3 +801,762 @@ func genFeeDelegatedCancelWithRatioTransaction() TxInternalData {
+				"gasPrice": "0x19",
+				"gas": "0xf4240",
+				"to": "0x7b65b75d204abed71587c9e519a89277766ee1d0",
+				"value": "0xa",
+				"from": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b",
+				"signatures": [{"V": "0x25", "R": "0xf3d0cd43661cabf53425535817c5058c27781f478cb5459874feaa462ed3a29a", "S": "0x6748abe186269ff10b8100a4b7d7fea274b53ea2905acbf498dc8b5ab1bf4fbc"}],
+				"hash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+			}`,
+		RpcJson: `{
+				"typeInt":8,
+				"type": "TxTypeValueTransfer",
+				"nonce": "0x4d2",
+				"gasPrice": "0x19",
+				"gas": "0xf4240",
+				"to": "0x7b65b75d204abed71587c9e519a89277766ee1d0",
+				"value": "0xa",
+				"signatures": [{"V": "0x25", "R": "0xf3d0cd43661cabf53425535817c5058c27781f478cb5459874feaa462ed3a29a", "S": "0x6748abe186269ff10b8100a4b7d7fea274b53ea2905acbf498dc8b5ab1bf4fbc"}]
+			}`,
+	},
+	{
+		Name: "09_FeeDelegatedValueTransfer", // kaia-sdk
+		Type: TxTypeFeeDelegatedValueTransfer,
+		Map: map[TxValueKeyType]interface{}{
+			TxValueKeyNonce:    uint64(1234),
+			TxValueKeyGasPrice: big.NewInt(0x19),
+			TxValueKeyGasLimit: uint64(0xf4240),
+			TxValueKeyTo:       common.HexToAddress("0x7b65B75d204aBed71587c9E519a89277766EE1d0"),
+			TxValueKeyAmount:   big.NewInt(10),
+			TxValueKeyFrom:     common.HexToAddress("0xa94f5374Fce5edBC8E2a8697C15331677e6EbF0B"),
+			TxValueKeyFeePayer: common.HexToAddress("0x5A0043070275d9f6054307Ee7348bD660849D90f"),
+		},
+		ChainID:         1,
+		SenderSigs:      []txSigHex{{v: 0x25, r: "0x9f8e49e2ad84b0732984398749956e807e4b526c786af3c5f7416b293e638956", s: "0x6bf88342092f6ff9fabe31739b2ebfa1409707ce54a54693e91a6b9bb77df0e7"}},
+		FeePayerSigs:    []txSigHex{{v: 0x26, r: "0xf45cf8d7f88c08e6b6ec0b3b562f34ca94283e4689021987abb6b0772ddfd80a", s: "0x298fe2c5aeabb6a518f4cbb5ff39631a5d88be505d3923374f65fdcf63c2955b"}},
+		SigRLP:          "0xf839b5f4098204d219830f4240947b65b75d204abed71587c9e519a89277766ee1d00a94a94f5374fce5edbc8e2a8697c15331677e6ebf0b018080",
+		SigFeePayerRLP:  "0xf84eb5f4098204d219830f4240947b65b75d204abed71587c9e519a89277766ee1d00a94a94f5374fce5edbc8e2a8697c15331677e6ebf0b945a0043070275d9f6054307ee7348bd660849d90f018080",
+		SenderTxHashRLP: "0x09f87a8204d219830f4240947b65b75d204abed71587c9e519a89277766ee1d00a94a94f5374fce5edbc8e2a8697c15331677e6ebf0bf845f84325a09f8e49e2ad84b0732984398749956e807e4b526c786af3c5f7416b293e638956a06bf88342092f6ff9fabe31739b2ebfa1409707ce54a54693e91a6b9bb77df0e7",
+		TxHashRLP:       "0x09f8d68204d219830f4240947b65b75d204abed71587c9e519a89277766ee1d00a94a94f5374fce5edbc8e2a8697c15331677e6ebf0bf845f84325a09f8e49e2ad84b0732984398749956e807e4b526c786af3c5f7416b293e638956a06bf88342092f6ff9fabe31739b2ebfa1409707ce54a54693e91a6b9bb77df0e7945a0043070275d9f6054307ee7348bd660849d90ff845f84326a0f45cf8d7f88c08e6b6ec0b3b562f34ca94283e4689021987abb6b0772ddfd80aa0298fe2c5aeabb6a518f4cbb5ff39631a5d88be505d3923374f65fdcf63c2955b",
+	},
+	{
+		Name: "10_FeeDelegatedValueTransferWithRatio", // kaia-sdk
```

Comment:
nit: Please write tx type in hex. (0a_FeeDelegatedValueTransferWithRatio) see https://github.com/kaiachain/kaia-sdk/tree/dev/ethers-ext/example/v6/transactions. Easier to see the pattern.